### PR TITLE
sync: `go get` everything at once

### PR DESCRIPTION
**Pull request by tamird**
Prevents a race condition on concurrent uses of `go get`.

@bdarnell @a-robinson @robfig

### CODE REVIEW

**Comment by tamird** [comment]
cc @raduberinde @clofresh @Viq111

**Comment by a-robinson** [comment]
You might want to leave a comment explaining why we're running `go get` on all packages in one command, but otherwise this looks reasonable and seems to fix the issue on my machine.

It's a shame that https://godoc.org/golang.org/x/tools/go/vcs didn't work for syncing to a revision.

**Comment by RaduBerinde** [comment]
+1 on the comment, reference the discussion in #34 
LGTM

File: sync.go [hunk 1]
```diff
@@ -125,37 +145,29 @@ func truncate(rev string) string {
...
 %s
 
 > import %s
 %s`, importPath, importPath, getStatus, importPath, err))
-		}
 	}
 
 	var maybeGot = ""
-	if bytes.Contains(getOutput, []byte("(download)")) {
+	if strings.Contains(getOutput, importPath+" "+"(download)") {
```
Comment (RaduBerinde) [inline sync.go]: why not just `+" (download)"`?

Comment (tamird) [inline sync.go]: because `getOutput` now contains the "download" line for every package, not just this one.

Comment (RaduBerinde) [inline sync.go]: I meant why not `importPath + " (download)"` instead of `importPath + " " + "(download)"`

**Comment by tamird** [comment]
Added a comment.

File: sync.go (hunk 1 above)
Comment (tamird) [inline sync.go]: Done.

**Comment by RaduBerinde** [comment]
LGTM

**Comment by Viq111** [comment]
I think I have some perf regression for our glockfile (see PR #34).

For a fresh `$GOPATH` for each test:
```
ᐅ go version
go version go1.6.2 darwin/amd64
/go/src/github.com/tamird/glock (go-get-race✔) ᐅ go install
ᐅ time glock sync ...
44.47s user 19.22s system 73% cpu 1:27.11 total
```


```
/go/src/github.com/DD/glock (master✔) ᐅ go install
ᐅ time glock sync ...
53.35s user 23.63s system 227% cpu 33.855 total
```

(^ This glock is the master branch with a revert of PR #34)

**Comment by tamird** [comment]
That's not surprising since `go get` is probably not parallel. Better to be correct than fast.

**Comment by robfig** [comment]
Thank you!

**Pull request merged**